cli.js

```diff
@@ -92,33 +95,33 @@ st.once('testserver', function (server) {
 
 	setState('download');
 	var ping = Math.round(server.bestPing);
-	stats.ping = (cli.flags.json) ? ping : chalk.cyan(ping + chalk.dim(' ms'));
+	stats.ping = cli.flags.json ? ping : chalk.cyan(ping + chalk.dim(' ms'));
 });
 
 st.on('downloadspeedprogress', function (speed) {
 	if (state === 'download' && cli.flags.json !== true) {
-		var download = roundTo(speed, speed > 10 ? 0 : 1);
-		stats.download = chalk.yellow(download + chalk.dim(' Mbps'));
+		var download = roundTo(speed * multiplier, speed > 10 ? 0 : 1);
```

Comment:
I don't think,
speed = speed * multiplier;
var download = roundTo(speed, speed > 10 ? 0 : 1); 
and 
var download = roundTo(speed * multiplier, speed > 10 ? 0 : 1);
are different.
In this case why can't we round it to 1 and 2 instead of 0 and 1. Your thoughts on this please? @sindresorhus @SamVerschueren